### PR TITLE
Feature/player cpu gen

### PR DESCRIPTION
**Pull request by crottsl7324**

### CODE REVIEW

File: source/GoFish.py [hunk 1]
```diff
@@ -27,26 +52,22 @@ def __init__(self,Players):
...
         while True:
             print(Players)
             #if player.deck = []
         #   - This is where were gonna have the game 
-        #   - logic go 
+        #   - logic go  a
 
 
-#Main function will handle only the starting of the program. 
+#Main function will hangit dle only the starting of the program.
```
Comment (NickBusman) [inline source/GoFish.py:64]: Little confused what "hangit dle" means

**Review by NickBusman — APPROVED** [review]

File: source/GoFish.py (hunk 1 above)
Comment (crottsl7324) [reply source/GoFish.py:64]: It was meant to say "handle" I believe.

**Review by davidcloar — APPROVED** [review]
Looks good.

**Review by crottsl7324 — COMMENTED** [review]
nice

**Pull request merged**